Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelDetectorElement.hpp

```diff
@@ -48,9 +51,8 @@ class GeoModelDetectorElement : public DetectorElementBase {
   /// @return a shared pointer to an instance of the detector element
   template <typename SurfaceType, typename BoundsType>
   static std::shared_ptr<GeoModelDetectorElement> createDetectorElement(
-      const GeoFullPhysVol& geoPhysVol,
-      const std::shared_ptr<BoundsType> bounds, const Transform3& sfTransform,
-      ActsScalar thickness) {
+      PVConstLink geoPhysVol, const std::shared_ptr<BoundsType> bounds,
```

Comment:
Is `PVConstLink` a typedef from GeoModel? I wonder if we should rather unroll that in ACTS since we usually want pointer types to be explicit.

Reply:
Yes it is from Geomodel.

Reply:
okay 👍 I guess there is not much we can do if GeoModel wants it this way